compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -2047,4 +2068,19 @@ private void validateDocAttribute(BLangDocumentationAttribute attribute) {
         }
         attribute.type = attributeSymbol.type;
     }
+
+    private void updateChildScopeNames(Stack<String> childScopes) {
```

Comment:
Can we write this method with, 
childScopes.replaceAll(s -> scopesMap.get(s));

Reply:
Fixed. Thanks.